zztin/ctimer
# Change vocal hint & Retrieve previous clock count for today!

(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies


## Description
**Describe what the change is**
Two major changes:
1. Update vocal hint so people don't forget to update results of the clock 
2. Fix bug #21: Clock count will accumulate throughout the whole day regardless if you close and restart the app. 

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to commandline, launch ctimer --debug
2. Check expected vocal hint behaviour after clock finishes.
3. Close the app, restart the app. 
4. Check "Done: ? clocks" <-- is it expected? 
5. Check ctimer.db file if clocks are stored correctly. 

## Related Issue
#21
